cookbook/security/multiple_guard_authenticators.rst

```diff
@@ -0,0 +1,168 @@
+How to Use Multiple Guard Authenticators
+========================================
+
+Guard authentication component allows you to easily use many different authenticators at a time.
+
+An entry point is a service id (of one of your authenticators) whose start()
+method should be called when an anonymous user hits a page that requires authentication.
+
+Multiple authenticators with shared entry point
+-----------------------------------------------
+
+Let's have an example of two authenticators: one based on login form, another one on facebook login.
+Both authenticators entry points redirect user to the same login page.
+However, in your configuration you have to explicitly say which entry point you want to use.
```

Comment:
We avoid the first person perspective in the docs. Thus, I would reword this paragraph to something like the following:

> Sometimes you want to offer your users different authentication mechanisms like a form login and a Facebook login while both entry points redirect the user to the same login page. However, [...]